### ISSUE TITLE
iOS - Home Screen - While using VoiceOver Screen Moved to the Left

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#121]
While using VoiceOver to read through the contents of the home screen, the screen moved to the left and up from the bottom for some reason. I was swiping right to read the next item when this happened. I would not expect the screen to move when using VoiceOver. 

There is extra white space on the right of the screen now, and the left is missing part of the DOL seal and part of the text below it which are highlighted in the screenshot.

Please see the attached screenshot.

![comply chain - ios- home screen - screen moved left when using voiceover](https://user-images.githubusercontent.com/10423487/74767728-68131400-5255-11ea-9423-b0944b74247f.png)

**Comment by jfc3-dol** [comment]
Adding one more screenshot.

![comply chain - ios- home screen - screen moved left and up when reading with voiceover on last item on screen](https://user-images.githubusercontent.com/10423487/74768525-bc6ac380-5256-11ea-95b8-46c60b19af8f.png)

**Comment by Abychalil** [comment]
Hi @jfc3-dol , 

Is this issue still there? We cannot recreate this from our end. I reinstalled the App and tried swiping (right and left)on the splash screen couple of times, I couldn't see any white space on the bottom or on the right side. Can you please retest this issue and comment?

**Comment by jfc3-dol** [comment]
@Abychalil,

I think it's no longer an issue. I will give it a check on our iOS devices (iPhone and iPad) tomorrow.

**Comment by jfc3-dol** [comment]
@Abychalil 

I tested this issue again with both our iPad Air 2 (13.3.1) and iPhone 7 (13.3.1) using VoiceOver. While swiping through the content, and it no longer moves the screen as previously.

So I'm going to close this issue.
